pop-api/src/v0/nfts.rs

```diff
@@ -320,3 +673,210 @@ impl From<PopApiError> for Error {
 		}
 	}
 }
+
+// Local implementations of pallet-nfts types
+mod types {
```

Comment:
Should this module be added to primitives?

Reply:
No, these are the ink! versions of the types in `pallet_nfts`, so that the types can be decoded without need a dependency on `pallet_nfts` in contracts.